mlflow/langchain/langchain_tracer.py

```diff
@@ -29,49 +32,56 @@
 VS_INDEX_DOC_URL_COL = "doc_uri"
 
 
+@dataclass
+class SpanWithToken:
```

Comment:
Will this class be used for Langchain flavor only? I feel this class will be used for other similar implementation and we may want to define it in a common space if this is the case.

Reply:
Great point, we will be using it for LlamaIndex / DSPy as well: https://github.com/mlflow/mlflow/pull/13793